frontend/tests/components/LoginModal.test.tsx

```diff
@@ -0,0 +1,46 @@
+import userEvent from "@testing-library/user-event";
+import { render, screen } from "tests/react-utils";
+
+import Header from "src/components/Header";
+import { LoginModal } from "src/components/LoginModal";
+
+const usePathnameMock = jest.fn().mockReturnValue("/fakepath");
+
+jest.mock("next/navigation", () => ({
+  usePathname: () => usePathnameMock() as string,
+}));
+
+jest.mock("src/hooks/useFeatureFlags", () => ({
+  useFeatureFlags: () => ({
+    checkFeatureFlag: () => true,
+  }),
+}));
+
+describe("LoginModal", () => {
```

Comment:
can we add a test that the login button directs to the correct location?

Reply:
This is a constant now, so it would be just checking the link. Can add in a follow up